Adding runCLI section to the Playground CLI page (#2583)
WordPress/wordpress-playground: packages/docs/site/docs/developers/05-local-development/04-wp-playground-cli.md

## Quickstart

> Running the Playground CLI is as simple as opening a command line and running:

Reviewer: The same nitpick about the word "simple" as below

## Need some help with the CLI?

> With the Playground CLI, you can use the `--help` flag to get a full list of available commands and arguments.

Reviewer: ```suggestion
With the Playground CLI, you can use the `--help` flag to get the full list of available commands and arguments.
```

```bash
npx @wp-playground/cli@latest --help
```

## Programmatic Usage with JavaScript

> The Playground CLI can also be controlled programmatically from your JavaScript/TypeScript code using the `runCLI` function. This gives you direct access to all CLI functionalities within your development environment, allowing you to enhance and automate your workflows. A key benefit is the ability to create robust end-to-end tests. Let's cover the basics of using `runCLI`.

Reviewer: Nitpick: We can remove some words and retain all the meaning:

```suggestion
The Playground CLI can also be controlled programmatically from your JavaScript/TypeScript code using the `runCLI` function. This gives you direct access to all CLI functionalities within your code, which is useful for automating end-to-end tests. Let's cover the basics of using `runCLI`.
```

You can provide a blueprint in two ways: either as an object literal directly passed to the `blueprint` property, or as a string containing the path to an external `.json` file.

```TypeScript
import { runCLI, RunCLIServer } from "@wp-playground/cli";

let cliServer: RunCLIServer;

cliServer = await runCLI({
  command: 'server',
  wp: 'latest',
  blueprint: {
    steps: [
        {
          "step": "setSiteOptions",
          "options": {
              "blogname": "Blueprint Title",
              "blogdescription": "A great blog description"
          }
        }
    ],
  },
});
```

For full type-safety when defining your blueprint object, you can import and use the `BlueprintDeclaration` type from the `@wp-playground/blueprints` package:

```TypeScript
import type { BlueprintDeclaration } from '@wp-playground/blueprints';

const myBlueprint: BlueprintDeclaration = {
  landingPage: "/wp-admin/",
  steps: [
    {
      "step": "installTheme",
      "themeData": {
        "resource": "wordpress.org/themes",
        "slug": "twentytwentyone"
      },
      "options": {
        "activate": true
      }
    }
  ]
};
```

### Mounting a plugin programmatically

> Simply start with It is possible to mount local directories programmatically using `runCLI`. The options `mount` and `mount-before-install` are available. The `hostPath` property expects a path to a directory on your local machine. This path should be relative to where your script is being executed..

Reviewer: One piece of feedback that keeps coming up is to avoid words like "simply" or "just" since it may not be simple for some folks. Let's stick to that.

Also, "Simply start with It is possible" seems like two different ways to start the same sentence :-) Also, there's a double `.` at the end of it.
